web/templates/tooltip_templates.hbs

```diff
@@ -19,8 +19,7 @@
     {{tooltip_hotkey_hints "R"}}
 </template>
 <template id="left_bar_compose_mobile_button_tooltip_template">
-    {{t 'New message' }}
-    {{tooltip_hotkey_hints "C"}}
+    {{t 'Start new conversation' }}
 </template>
```

Comment:
This change shouldn't be part of this PR.

Reply:
I think because of that the test were failing preivously. That's why I have indented it.

Reply:
We're not talking about why the testcases were failing. These changes shouldn't be in this PR as it's indepent of the issue you're trying to work on.